packages/playground-examples/copy/ja/3-7/Fixits/Infer From Usage Changes.ts

```diff
@@ -0,0 +1,32 @@
+//// { compiler: {  noImplicitAny: false }, order: 2 }
+
+// 3.7 で TypeScript 既存の「使用状況からの推論」によるコード修正はよりスマートになりました。
+// 既知の重要な型 (string, number, array, Promise) のリストを用いて、
```

Comment:
IMO: かなり意訳ですが、このknownは一般的に知られているぐらいの意味だと思うので、考えられているにしてみました
```suggestion
// 重要だと考えられている型 (string, number, array, Promise) のリストを用いて、
```